src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/AzureBlobStorageCacheOptionsConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Extensions.Options;
+using SixLabors.ImageSharp.Web.Caching.Azure;
+
+namespace OrchardCore.Media.Azure.Services;
+
+// Configuration for ImageSharp's own configuration object. We just pass the settings from the Orchard Core
+// configuration.
+internal class AzureBlobStorageCacheOptionsConfiguration : IConfigureOptions<AzureBlobStorageCacheOptions>
```

Comment:
What value does `AzureBlobStorageCacheOptions` add here? It seems to be a proxy of `ImageSharpImageCacheOptions`. Can't we just use `ImageSharpImageCacheOptions` instead of having to add a new options class?

Reply:
This configures `AzureBlobStorageCacheOptions`, which is the IS Azure cache options class. `ImageSharpImageCacheOptions` is the OC options class with more options (and in a slightly different way, like having `BasePath` instead of `CacheFolder`, so it's in sync with other such OC configs). Both are needed; we expose the OC options only, and then fill the IS ones behind the scenes.